csrc/ir/internal_nodes.h

```diff
@@ -1359,16 +1359,35 @@ class GroupedWelfordOp : public Expr {
 class NVF_API MmaOp : public Expr {
  public:
   using AxesData = std::vector<int64_t>;
+  // AxisMapping denotes the pairing of two input dimensions to produce an
+  // output dimension. It holds two vectors of integers indicating the
+  // corresponding position of each output axis in either the A or B input.
+  // Positions are absolute and refer to the noReductions logical domain. NOTE:
+  // -1 indicates that the axis does not exist, so Broadcast and Reduction
+  // dimensions should not have position -1.
```

Comment:
Could you provide an example? For example, what does it mean if we have:
```C++
a_axes = {0, 2, 1, 3}
b_axes = {1, 0 , 3, 2}
```
does it mean `a.axis(0)` is mapped to `b.axis(1)`?

Reply:
I'll put this in a comment but yes, this would mean we would map the following sets:
```
{a.axis(0), b.axis(1), out.axis(0)}
{a.axis(2), b.axis(0), out.axis(1)}
{a.axis(1), b.axis(3), out.axis(2)}
{a.axis(3), b.axis(2), out.axis(3)}
```